pkg/environment/environment.go

```diff
@@ -346,6 +363,43 @@ func getEnvironmentFromParams(params kernel.Params) (Environment, error) {
 		env.RegistrarURL = registrar
 	}
 
+	if geoip := config.GeoipURLs; len(geoip) > 0 {
+		env.GeoipURLs = geoip
+	}
+
+	// flist url and hub storage urls shouldn't listen to changes in config as long as we can't change it at run time.
```

Comment:
yes, but if we faced samething when all *.tf were blocked then we will not be able to boot new machines and all those configurable urls will not help unless we can connect to the new storage url